audio_video_recorder/RecordTask task
---
bool success
string message
